tests/orm/utils/test_managers.py

```diff
@@ -164,10 +168,20 @@ def test_link_manager_with_nested_namespaces(clear_database_before_test):
         assert calc.inputs.nested__sub__namespace.uuid == inp1.uuid
         assert calc.outputs.nested__sub__namespace.uuid == out1.uuid
 
+    # Dunder methods should not invoke the deprecation warning
+    with pytest.warns(None) as record:
+        try:
+            calc.inputs.__name__
+        except AttributeError:
+            pass
+        assert not record
+
     # Must raise a AttributeError, otherwise tab competion will not work
-    with pytest.raises(AttributeError):
-        _ = calc.outputs.nested.not_existent
+    for attribute in ['not_existent', 'not__existent__nested', 'remote_folder__namespace']:
```

Comment:
I guess you should separate `'not__existent__nested'` and `'remote_folder__namespace'` from `'not_existent'`. They are not nested in `nested` namespace.

Reply:
Good catch. This was also the reason that the `TypeError` catch block was not being hit in the tests. It should be fixed now.